app/lib/clients/vault/entity.rb

```diff
@@ -8,6 +8,49 @@ def put_entity(name, policies, metadata = {})
                              metadata: metadata)
       end
 
+      def put_group(name, policies, auth_method = "oidc", metadata = {})
+        Domain.with_advisory_lock(name) do
+          original_policies, original_metadata = get_group_data(name)
+          policies = (policies || []) + original_policies
+          metadata = metadata.merge((original_metadata || {}))
+          client.logical.write("identity/group",
+                               {
+                                 name: name,
+                                 type: "external",
```

Comment:
we shouldn't assume this is "external".  I think it is ok for now to throw an exception if it is not external, but I think it is dangerous to just overwrite it without checking.

Reply:
okay will ponder this -- maybe best to read and patch only policies if previously extant

Reply:
now external by default but takes the extant value where it exists